core-java-modules/core-java-exceptions/pom.xml

```diff
@@ -0,0 +1,26 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
We will also need to add this module to the modules section in the core-java-modules/pom.xml

Reply:
> We will also need to add this module to the modules section in the core-java-modules/pom.xml

Done